src/options.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable import/prefer-default-export */
+
+// TODO: ask 'schema-utils' or 'webpack' to export SchemaType
+import { validate } from 'schema-utils';
+
+export const SCHEMA: Parameters<typeof validate>[0] = {
```

Comment:
add a comment to explain what this is all about

Reply:
Done